gns3fy/gns3fy.py

```diff
@@ -1897,3 +1900,47 @@ def restore_snapshot(self, name=None, snapshot_id=None):
 
         # Update the whole project
         self.get()
+
+    def arrange_nodes_circular(self, radius=120):
+        """
+        Re-arrgange the existing nodes
+        in a circular fashion
+
+        **Attributes:**
+
+        - project instance created
+
+        **Example**
+
+        ```python
+        >>> proj = Project(name='project_name', connector=Gns3connector)
+        >>> proj.arrange_nodes()
+        """
+
+        self.get()
+        if self.status != "opened":
+            self.open()
+
+        _r = 120
+        # radius
+        _angle = (2 * pi) / len(self.nodes)
+        # The Y Axis is inverted in GNS3, so the -Y is UP
+        for index, n in enumerate(self.nodes):
+            _x = int(_r * (sin(_angle * index)))
+            _y = int(_r * (-cos(_angle * index)))
+            n.update(x=_x, y=_y)
+
+    def get_drawings(self):
+        """
+        Retrieves list of drawins  of the project
+        **Required Project instance attributes:**
+        - `project_id`
+        - `connector`
+        """
+        self._verify_before_action()
+        self.get()
```

Comment:
Is the `self.get` really needed? If you look at the `get_snapshots` is not there. One thing we are trying to do is be cheap on the calls made to the API